doc/release_notes.rst

```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
 * The ACRN Block, NIC, and console Virtio drivers
```

Comment:
These three lines (all three about virtio) really feel weird to me. Are these really 3 SW components? I would have called out the hypervisor, device model, tools and acrn-kernel instead. Perhaps even highlighting GVT-g in this release there.

Reply:
That was from the previous release notes.  I'm just going to delete that section.